### ISSUE TITLE
how to modify taskweaver if my plugin is writeen in c++ language, and I need the llm generate the c++  code and call the c++ plguin

### ISSUE THREAD

**Issue by pily1** [microsoft/TaskWeaver#428]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

"How do I modify Taskweaver if my plugin is written in C++ and I need LLVM to generate the C++ code and call the C++ plugin?
 What steps should I follow to change from generated Python code to C++ code?"

**Comment by liqul** [comment]
No, I don't think C++ could be supported here as the backend engine is only for Python. If you really need to exploit your C++ code, I think the only way is to somehow wrap it as something callable from Python.
